xbmc/guilib/TextureManager.cpp

```diff
@@ -297,6 +298,24 @@ const CTextureArray& CGUITextureManager::Load(const std::string& strTextureName,
   if (!HasTexture(strTextureName, &strPath, &bundle, &size))
     return emptyTexture;
 
+  unsigned int textureFormat = XB_FMT_A8R8G8B8;
+  if (StringUtils::StartsWith(strTextureName, "special://skin/"))
+  {
+    std::vector<std::string> s = StringUtils::Split(strTextureName, ".");
```

Comment:
What do we pay you by, the character? :stuck_out_tongue_winking_eye: No one ever underappreciates a good explanatory variable name.

Reply:
Done.